docker/environment/Dockerfile

```
@@ -40,4 +41,4 @@ ENV ZKSYNC_HOME=/usr/src/zksync
 ENV PATH="${ZKSYNC_HOME}/bin:${PATH}"
 ENV CI=0
 RUN cargo install sccache
-ENV RUSTC_WRAPPER=/usr/local/cargo/bin/sccache
+ENV RUSTC_WRAPPER=/usr/local/cargo/bin/sccache
```

I added an empty line ad the end
```suggestion
ENV RUSTC_WRAPPER=/usr/local/cargo/bin/sccache

```